internal/app/frontend/frontend_service_test.go

```diff
@@ -32,48 +32,67 @@ import (
 )
 
 func TestDoCreateTickets(t *testing.T) {
-	store, closer := statestoreTesting.NewStoreServiceForTesting(t, viper.New())
+	cfg := viper.New()
+	cfg.Set("playerIndices", []string{"test-property"})
+	store, closer := statestoreTesting.NewStoreServiceForTesting(t, cfg)
 	defer closer()
 
 	normalCtx := context.Background()
 	cancelledCtx, cancel := context.WithCancel(context.Background())
 	cancel()
 
-	testTicket := &pb.Ticket{
+	numTicket := &pb.Ticket{
 		Properties: &structpb.Struct{
 			Fields: map[string]*structpb.Value{
 				"test-property": {Kind: &structpb.Value_NumberValue{NumberValue: 1}},
 			},
 		},
 	}
 
+	stringTicket := &pb.Ticket{
+		Properties: &structpb.Struct{
+			Fields: map[string]*structpb.Value{
+				"test-property": {Kind: &structpb.Value_StringValue{StringValue: "string"}},
+			},
+		},
+	}
+
 	tests := []struct {
 		description string
+		ticket      *pb.Ticket
 		ctx         context.Context
 		wantErr     bool
 	}{
+		{
+			description: "expect precondition error since open-match does not support properties other than number",
+			ctx:         normalCtx,
+			ticket:      stringTicket,
+			wantErr:     true,
+		},
 		{
 			description: "expect error with canceled context",
 			ctx:         cancelledCtx,
+			ticket:      numTicket,
 			wantErr:     true,
 		},
 		{
 			description: "expect normal return with default context",
 			ctx:         normalCtx,
+			ticket:      numTicket,
 			wantErr:     false,
 		},
 	}
 
 	for _, test := range tests {
 		t.Run(test.description, func(t *testing.T) {
-			res, err := doCreateTicket(test.ctx, &pb.CreateTicketRequest{Ticket: testTicket}, store)
+			res, err := doCreateTicket(test.ctx, &pb.CreateTicketRequest{Ticket: test.ticket}, store)
 
 			assert.Equal(t, test.wantErr, err != nil)
```

Comment:
You may want to check the actual failure code too. Always a good practice to validate that it failed with the code you expected it to fail and not something else.